src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/McpServer/McpServer-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Common", "AI", "MCP" ],
+  "identity": "Microsoft.Extensions.AI.Templates.McpServer.CSharp",
+  "name": "Local MCP Server Console App",
+  "description": "A project template for creating a local MCP server using the .NET MCP SDK.",
```

Comment:
```suggestion
  "description": "A project template for creating a local MCP server using the MCP C# SDK.",
```

Reply:
Seems fine to me. For my learning, could you explain why we don't call it ".NET"? Could this work for F# theoretically? (I noticed there is a `-CSharp` naming convention for templates so theoretically we could maybe make an FSharp MCP Server template using this SDK? Not sure.